versioned_docs/version-1.2.0/01-get-started.md

```diff
@@ -237,6 +237,6 @@ Working with a database is an extensive subject. Learn more in the [Database](co
 :::
 
 ## Where to go next
-You should now have a basic understanding of how Serverpod works. The different topics are described in more details in the _Concepts_ section of the documentation. If you are unfamiliar with server-side development, a good starting place for learning is to do the [Build your first app](tutorials/first-app) tutorial. There are also many good video tutorials linked in the _Tutorials_ section.
+You should now have a basic understanding of how Serverpod works. The different topics are described in more detail in the _Concepts_ section of the documentation. If you are unfamiliar with server-side development, a good staring place for learning is to do the [Build your first app](tutorials/first-app) tutorial. There are also many good video tutorials linked in the _Tutorials_ section.
```

Comment:
Suggestion: The change to "staring" here seems wrong.

Reply:
Good catch! will fix! 👍